samples/unit-tests/axis/ordinal/demo.js

```diff
@@ -355,9 +355,11 @@ QUnit.test('lin2val- unit test for values outside the plotArea.', function (asse
         },
         series: [{
             points: [{
+                isInside: true, // #18459
                 x: 3,
                 plotX: -20
             }, {
+                isInside: true, // #18459
```

Comment:
There is no such option... Only the internal, computed property `Point.isInside`. So is this test realistic, or are we cheating by forcing a property that should be computed?

Reply:
Sure thing, there is no such property - but since we don't even create a chart there, and use fake points, this property won't get computed (right?). We test `lin2val` outside the chart environment. Therefore I decided to manually put it in there, to ensure that a new logic will treat those points as they are inside the chart.

In fact, I think line 358 should be removed here.

Reply:
Ok, I didn't realize it was mock points.